src/pg/brc20/brc20-pg-store.ts

```diff
@@ -268,6 +314,18 @@ export class Brc20PgStore {
         INSERT INTO brc20_balances ${sql(changes)}
         ON CONFLICT ON CONSTRAINT brc20_balances_inscription_id_type_unique DO NOTHING
       `;
+
+      // Add to event history
+      const event: DbBrc20EventInsert = {
+        operation: 'transfer_send',
+        inscription_id: transfer.inscription_id,
+        genesis_location_id: args.id,
+        brc20_deploy_id: transfer.brc20_deploy_id,
+        deploy_id: null,
+        mint_id: null,
+        transfer_id: transfer.id,
+      };
+      await this.parent.sql`INSERT INTO brc20_events ${this.parent.sql(event)}`;
```

Comment:
Can you integrate this INSERT into the INSERT above? (Using a common table expression)
This would be a considerable performance boost